pymedphys/tests/dicom/test_anonymise.py

```diff
@@ -31,10 +32,9 @@
 from pymedphys._dicom.utilities import remove_file
 from pymedphys.dicom import anonymise as anonymise_dataset
 
-HERE = dirname(abspath(__file__))
-DATA_DIR = pjoin(HERE, "data", "anonymise")
-TEST_FILEPATH = pjoin(DATA_DIR, "RP.almost_anonymised.dcm")
-
+DATA_PATHS = download.zip_data_paths("rtplan-anonymisation.zip")
+TEST_POSIX_PATH = next(x for x in DATA_PATHS if x.name == "RP.almost_anonymised.dcm")
+TEST_FILEPATH = TEST_POSIX_PATH.absolute().as_posix()
```

Comment:
Could this be why it's failing on Windows?

Reply:
yes.
I'll try str() instead of as_posix().  It's not clear from the documentation for zip_data_paths what gets returned other than it being pathlib related, but there's a hierarchy of classes that are possible.
Once I have the windows test working, I'll go back in and see about using a fixture or lru_cache.
So I won't expect a merge until I've got that worked out.

Reply:
The `zip_data_paths` function returns a list containing all of the extracted paths that were found within the zip as the same type returned by `pathlib.Path`, this will be a Windows path on Windows systems, and a Posix path on Mac and Linux.